File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.js
function solution(num_list) {
    let answer = new Array(num_list.length);
    let count = 0;
    for(let i = num_list.length - 1; i >= 0; i--) {
        answer[count] = num_list[i];
        count++;
    }
    return answer;
}

File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.c
int* solution(int* num_list, int length) {
    int* answer = (int*)malloc(length * sizeof(int));
    int count = 0;
    for(int i = length - 1; i >= 0; i--) {
        answer[count] = num_list[i];
        count++;
    }
    return answer;
}